decidim/decidim
# Add malformed file errors when CSV reading fails

#### :tophat: What? Why?
At the admin panel when we upload specific CSV files and they are malformed (e.g. wrong encoding), the system will just show a general HTTP 500 error for the user which can be confusing.

This adds proper error messages in these situations.

#### :pushpin: Related Issues
- Fixes #7240

#### Testing
- Go to the participatory space private user import from CSV view
- Import the file added in this PR (import_participatory_space_private_users_iso8859-1.csv) as is
- See the error on the page without an exception

### :camera: Screenshots
![Malformed CSV error](https://user-images.githubusercontent.com/864340/181052543-be298901-6d46-4836-a85c-24d9375c1c58.png)
